File: frontend/src/constants/CustomerRequisite.ts
import { Dayjs } from "dayjs";

export type CustomerRequisite = {
  
  id: number;
  paymentAccount: string;
  bank: string;
  bik: string;
  organizationId: number;
};


export type CustomerRequisiteCreateModel = {
  
  id: number;
  paymentAccount: string;
  bank: string;
  bik: string;
  organizationId: number;
};

File: backend/Domain/Entities/CustomerRequisite.cs

namespace Domain.Entities
{
    public class CustomerRequisite : BaseLogDomain, IBaseDomain
    {
        public int Id { get; set; }
		public string PaymentAccount { get; set; }
		public string Bank { get; set; }
		public string Bik { get; set; }
		public int OrganizationId { get; set; }
		
    }
}